### DISCUSSION TITLE
UI bug on account domains page

### DISCUSSION THREAD

**Discussion by ikaro-souza** [vercel/community#6936] [Bug Report] [answered]
### Page to Investigate

https://vercel.com/account/domains

### Steps to Reproduce

Open the /account/domains page.

There's an icon being rendered right under the search input, as shown in the image below.

![ui bug](https://github.com/vercel/community/assets/44532346/0110623d-222a-420d-86a2-1dfafb0adf96)

**Comment by swarnava** [comment, marked as answer]
Hey, thanks for reporting. I have forwarded to our Product team. :)
